File: src/Level1/서울에서김서방찾기.java
package Level1;
import java.util.*;
import java.util.stream.Collectors;

public class 서울에서김서방찾기 {
    public String solution(String[] seoul) {
        return "김서방은 " +
                Arrays.stream(seoul)
                        .collect(Collectors.toList())
                        .indexOf("Kim") +
                "에 있다";
    }
}

File: src/Level1/서울에서김서방찾기.kt
fun main(){
    println(solution(arrayOf("Jane", "Kim")))
    println(solution1(arrayOf("Jane", "Kim")))
}

private fun solution(seoul: Array<String>): String{
    seoul.forEachIndexed{ i, v -> if(v == "Kim") return "김서방은 ${i}에 있다" }
    return ""
}

private fun solution1(seoul: Array<String>) = "김서방은 ${seoul.indexOf("Kim")}에 있다"